src/vstart.sh

```diff
@@ -1017,7 +1017,8 @@ do_rgw()
     i=0
     for rgw in j k l m n o p q r s t u v; do
         echo start rgw on http${CEPH_RGW_HTTPS}://localhost:$((CEPH_RGW_PORT_NUM + i))
```

Comment:
might want to update the this line to use `$current_port` also.

Reply:
Addressed.